src/main/java/com/algorand/algosdk/v2/client/model/MiniAssetHolding.java

```diff
@@ -38,6 +38,20 @@ public String address() throws NoSuchAlgorithmException {
     @JsonProperty("is-frozen")
     public Boolean isFrozen;
 
+    /**
+     * (apan) defines the what additional actions occur with the transaction.
+     * Valid types:
+     *   noop
+     *   optin
+     *   closeout
+     *   clear
+     *   update
+     *   update
+     *   delete
+     */
+    @JsonProperty("OnCompletion")
+    public Enums.OnCompletion onCompletion;
```

Comment:
@Eric-Warehime and other reviewers - Sanity checking to confirm the `onCompletion` diff is expected?

From my inspection, it seems _unexpected_ though I still know little about API generation.
* Here's the indexer commit that I presume was referenced to generate the diff.    https://github.com/algorand/indexer/blob/0e409876e8def1de37791d6968320f0cf7da1ec5/api/indexer.oas2.json#L1637-L1671.
* It seems `MiniAssetHolding` does _not_ have `OnCompletion` as a field.  
* Curiously, it's the next type described.  Unclear if it's coincidental.

Reply:
Thanks for the callout, my previous generator changes were incorrectly associating enums with whichever model definition was alphabetically prior to it. 
Updates in the generator should now allow each language to take (or not take) a separate action on enum definitions allowing Java to store them in `Enums.java` regardless of their inclusion in paths, queries, etc.